routers/api/v1/repo/issue_tracked_time.go

```diff
@@ -203,15 +206,58 @@ func ListTrackedTimesByRepository(ctx *context.APIContext) {
 	//   description: name of the repo
 	//   type: string
 	//   required: true
+	// - name: user
+	//   in: query
+	//   description: optional filter by user
+	//   type: string
+	// - name: since
+	//   in: query
+	//   description: Only show times updated after the given time. This is a timestamp in ISO 8601 format
+	//   type: strfmt date-time
```

Comment:
You need to use type: string and then add a format: date-time.

You will need to accept RFC 3339 encoded date times as per:

https://swagger.io/docs/specification/data-models/data-types/#string

date-time – the date-time notation as defined by RFC 3339, section 5.6, for example, 2017-07-21T17:32:28Z

Reply:
```suggestion
	//   type: string
	//   format: date-time
```